stylance-cli/Cargo.toml

```diff
@@ -20,9 +20,7 @@ stylance-core = { workspace = true }
 anyhow = "1.0.79"
 
 clap = { version = "4.4.12", features = ["derive", "cargo"], optional = true }
-notify = { version = "6.1.1", default-features = false, features = [
-	"macos_kqueue",
-], optional = true }
+notify = { version = "6.1.1", optional = true }
```

Comment:
6.1.1 docs warn about an issue that happens when mixing the default feature crossbeam-channel, which is why default-features was disabled and macos_kqueue was enabled manually.

I'm not sure if this issue will affect stylance or not, but I'd either update to 8.2.0 or disable the default features and enable macos_fsevent manually.

Reply:
Done!